scripts/doc-checker.py

```diff
@@ -22,7 +22,7 @@
 
```

Comment:
I don't see any `sys.path` modifications I've suggested.
It means that to run the script correctly the developer would have to install `modin` from its development directory, or you're risking to validate some stale version which just happened to be installed in the environment.

Reply:
Yes, but `pip install -e .[all]` also install modin. I didn't see a way to disable modin installation but the dependencies installed in the way are needed.

Reply:
This is irrelevant. What I'm trying to get is to do two things for a developer running this script locally (not on GitHub CI):

1. Make sure the script is validating correct sources, not something random that was installed ages ago (remember that a developer's environment isn't a shiny new clean CI one)
2. Allow the developer to _not_ install Modin whatsoever (I, for once, am running it using `$PYTHONPATH`).

Reply:
Ok, I got it